src/video-player.js

```diff
@@ -24,6 +24,7 @@ import { extendCloudinaryConfig, normalizeOptions, isRawUrl } from './plugins/cl
 // #if (!process.env.WEBPACK_BUILD_LIGHT)
 import qualitySelector from './components/qualitySelector/qualitySelector.js';
 import { interactionAreaService } from './components/interaction-area/interaction-area.service';
+import { getProfile } from './components/profiles/profiles';
```

Comment:
this means it will not be available in the light build, was that the intention?

Reply:
changed to be lazy loaded so not relevant